great_expectations/validator/validation_graph.py

```diff
@@ -39,11 +40,11 @@ def metric_value_kwargs(self):
 
     @property
     def metric_domain_kwargs_id(self):
-        return self._metric_domain_kwargs.to_id()
+        return self.metric_domain_kwargs.to_id()
 
     @property
     def metric_value_kwargs_id(self):
-        return self._metric_value_kwargs.to_id()
+        return self.metric_value_kwargs.to_id()
 
```

Comment:
Why do we need to change the accessibility of these attributes? I think they can remain private (especially since we can access them using the `@property` decorator) but can you let me know your reasoning here?

Reply:
@cdkini For better maintenance, I changed the internal use to call the already define accessor methods (they were there before).  Otherwise, there is a risk of a typo, causing different outcomes.  This way, if the accessor changes, all uses that need the private property will change the same way.